docs/Permissions/Onchain-Permissioning/Onchain-Permissioning.md

```diff
@@ -0,0 +1,53 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node, account, and admin account whitelists. 
+Using onchain permissioning enables all nodes to read the whitelists from one location.                                        
+
+## Permissioning Contracts
+
+The permissioning smart contracts are provided in the [PegaSysEng/permissioning-smart-contracts](https://github.com/PegaSysEng/permissioning-smart-contracts) repository: 
+
+* Ingress contracts for nodes and accounts - simple contracts functioning as gateways to the Admin, Node Rules, 
```

Comment:
I wouldn't use the words simple contract here...
Maybe:
`proxy contracts that defer the permissioning logic to the Node and Account rules contracts.`

btw: the ingress contract doen't know anything about the admins :)

Reply:
Fixed